packages/sit-onyx/src/components/OnyxNavItem/OnyxNavItem.ct.tsx

```diff
@@ -6,7 +6,7 @@ test.describe("Screenshot tests", () => {
   executeMatrixScreenshotTest({
     name: "NavItem",
     columns: ["default", "active"],
-    rows: ["default", "hover", "focus-visible"],
+    rows: ["default", "hover", "focus-visible", "external link"],
```

Comment:
```suggestion
    rows: ["default", "hover", "focus-visible", "external-link"],
```

rows and columns must not have spaces because the names are used as CSS grid-area which breaks when using spaces